examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,626 @@
+    "def summarize(text: str):\n",
+    "    pass\n",
+    "\n",
+    "\n",
+    "@openai.call(\"gpt-4o-mini\")\n",
+    "@prompt_template(\"Translate the following to French: {text}\")\n",
+    "def translate_to_french(text: str):\n",
+    "    pass\n",
+    "\n",
+    "\n",
+    "original_text = \"Recently, it's been really hot, so I’m thinking of gathering with friends to enjoy some champagne outdoors. I’m looking for something refreshing and light, especially a champagne that’s enjoyable even on a hot day. If possible, I’d love to know about any recommended champagne that’s perfect for relaxing with everyone.\"\n",
+    "summary = summarize(original_text)\n",
+    "french_summary = translate_to_french(summary.content)\n",
+    "print(french_summary.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This chain first summarizes a text, then translates the summary to French, demonstrating how you can combine multiple LLM calls to perform more complex tasks."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## JSON Mode\n",
+    "\n",
+    "JSON mode allows you to directly parse LLM outputs as JSON. This is useful when you need structured data output.\n",
+    "\n",
+    "Here's an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@openai.call(\"gpt-4o-mini\", json_mode=True)\n",
+    "@prompt_template(\"Provide information about {city} in JSON format\")\n",
+    "def city_info(city: str):\n",
+    "    pass\n",
+    "\n",
+    "\n",
+    "response = city_info(\"Tokyo\")\n",
+    "print(response.content)  # This will be a JSON-formatted string"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By setting `json_mode=True`, we instruct the LLM to format its response as JSON, which can then be easily parsed and used in your application."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Output Parsers\n",
+    "\n",
+    "Output parsers allow you to process LLM responses in custom formats. This is useful when you need to extract specific information or transform the output in a particular way.\n",
+    "\n",
+    "Here's an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class CountryInfo(BaseModel):\n",
+    "    country: str\n",
+    "    capital: str\n",
+    "\n",
+    "\n",
+    "def parse_country_and_capital(response: openai.OpenAICallResponse) -> CountryInfo:\n",
+    "    country, capital = response.content.split(\" -> \")\n",
+    "    return CountryInfo(country=country, capital=capital)\n",
+    "\n",
+    "\n",
+    "@openai.call(\"gpt-4o-mini\", output_parser=parse_country_and_capital)\n",
+    "@prompt_template(\"What is the capital of {country}? the format 'country -> capital'.\")\n",
```

Comment:
nit: update "the format ..." to "Your output MUST follow the format 'country -> capital'."